css/styles.css

```diff
@@ -0,0 +1,231 @@
+* {
+    box-sizing: border-box;
+    --basicColor: #ff883e;
+}
+
+.categories {
+    border-top: 1px solid lightgrey;
+    border-bottom: 1px solid lightgrey;
+    list-style: none;
+    margin-top: 50px;    
+    clear: both;
+}
+
+.categories li {  
+    list-style-type:none; 
+    padding: 5px 25px;
+}
+
+.categories a,
+.categories a:link,
+.categories a:visited {
+    color: black;
+    text-decoration: none;
+}
+
+.categories a:hover{
+    text-decoration: underline;
+}
+
+h3 {
+    text-align: center;
+}
+
+header {
+    display: inline-block;
+    width: 100%;
+}
+
+
+footer {
+    max-height: 20px;
+    font-size: 15px;
+    margin-top: 20px;
+    margin-bottom: 10px;
+    border-top: 1px solid lightgrey;
+    padding: 10px;
+}
+
+footer span {
+    font-style: italic;
+
+}
+
+footer a,
+footer a:link,
+footer a:visited {
+    color: black;
+    text-decoration: none;
+}
+
+footer a:hover{
+    text-decoration: underline;
+}
+
+
+.firstSection div img {    
+    float: right;
+    margin-right: 50px;
+    margin-top: 19px;
+    width: 600px;
+    height: 300px;
+}
+
+
+.firstSection div button {
+    margin-top: 20px; 
+    margin-right: 50px;
+    padding: 35px 245px;
+    border: none;
+    background-color: #26547c;
+    color: white;
+    float: right;
+}
+
+.secondSection ul,
+.thirdSection ul {
+    list-style-type: none;
+    columns: 3;
```

Comment:
Will be better to do the colums with flex or grid.
Colums is intented to be used with long text and not for elements